db/migrate/20240405184242_create_orgs_reps_tables.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+class CreateOrgsRepsTables < ActiveRecord::Migration[7.1]
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
+  def change
+    create_table :accredited_attorneys, id: false, primary_key: :registration_number do |t|
+      t.string :registration_number, null: false
```

Comment:
Thoughts on using the OGC uuid as the id?

Reply:
Right!